### ISSUE TITLE
Babel 7

### ISSUE THREAD

**Issue by jaredpalmer** [jaredpalmer/razzle#764]

**Comment by stigkj** [comment]
Could this be closed? Or is it waiting for the release?

**Comment by krazyjakee** [comment]
In my opinion, since it's in the v4 alpha, I think it should be closed.

**Comment by adaam2** [comment]
@jaredpalmer Hey I've got an existing Razzle project that evidently was bootstrapped with a pre Babel 7 version of Razzle. I'm trying to setup Storybook which as others have mentioned has a hard dep on Babel 7 (e.g. `@babel/core`).

I'm trying to use the Babel upgrade tool (`npx babel-upgrade --write --install`) but I am running into the below error as Razzle is fixed to `"babel-core": "6.25.0"` in `master`:

```
web_1      | ERROR in ./src/client.js
web_1      | Module build failed (from ./node_modules/razzle/node_modules/babel-loader/lib/index.js):
web_1      | Error: Requires Babel "^7.0.0-0", but was loaded with "6.25.0". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel.
... etc ...
```

Could someone provide some docs on how to upgrade an existing Razzle project to use Babel 7 (my preferred package.json is below)

```

{
  "name": "my-razzle-app",
  "version": "0.1.0",
  "license": "MIT",
  "scripts": {
    "start": "razzle start",
    "build": "razzle build",
    "test": "razzle test --env=jsdom",
    "start:prod": "NODE_ENV=production node build/server.js",
    "storybook": "start-storybook"
  },
  "dependencies": {
    "apollo-server-express": "^2.4.8",
    "express": "^4.16.4",
    "graphql": "^14.2.1",
    "grommet": "^2.6.5",
    "mongoose": "^5.4.22",
    "morgan": "^1.9.1",
    "razzle": "^2.4.1",
    "react": "^16.8.6",
    "react-dom": "^16.8.6",
    "react-router-dom": "^5.0.0",
    "styled-components": "^4.2.0"
  },
  "devDependencies": {
    "@babel/core": "^7.4.3",
    "@babel/plugin-proposal-class-properties": "^7.0.0",
    "@babel/plugin-proposal-decorators": "^7.0.0",
    "@babel/plugin-proposal-export-namespace-from": "^7.0.0",
    "@babel/plugin-proposal-function-sent": "^7.0.0",
    "@babel/plugin-proposal-json-strings": "^7.0.0",
    "@babel/plugin-proposal-numeric-separator": "^7.0.0",
    "@babel/plugin-proposal-throw-expressions": "^7.0.0",
    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
    "@babel/plugin-syntax-import-meta": "^7.0.0",
    "@babel/plugin-transform-modules-systemjs": "7.4.0",
    "@babel/plugin-transform-new-target": "7.4.0",
    "@babel/plugin-transform-shorthand-properties": "7.2.0",
    "@babel/plugin-transform-template-literals": "7.2.0",
    "@babel/plugin-transform-unicode-regex": "7.4.3",
    "@babel/plugin-transform-runtime": "7.4.0",
    "@babel/preset-env": "^7.0.0",
    "@babel/preset-react": "^7.0.0",
    "@storybook/react": "^5.0.6",
    "babel-eslint": "^10.0.1",
    "babel-loader": "^8.0.5",
    "eslint": "^5.16.0",
    "eslint-config-airbnb": "^17.1.0",
    "eslint-plugin-import": "^2.16.0",
    "eslint-plugin-jsx-a11y": "^6.2.1",
    "eslint-plugin-react": "^7.12.4",
    "webpack": "^4.29.6"
  }
}
```

I guess I could check out the `next` branch of Razzle but don't know how to use it to upgrade an existing Razzle project?
